frontend/src/container/LogDetailedView/Overview.tsx

```diff
@@ -53,7 +53,6 @@ function Overview({
 			enabled: false,
 		},
 		fontWeight: 400,
-		// fontFamily: 'SF Mono',
 		fontFamily: 'Space Mono',
```

Comment:
We wanted to move to Geist Mono. Check if it's already included and we can use it

Reply:
@ahmadshaheer have we included the geist mono font ?

Reply:
@vikrantgupta25, Yeah, it's on this [PR](https://github.com/SigNoz/signoz/pull/5347), but it needs to be reviewed :slightly_smiling_face: . 

CC: @YounixM

Reply:
let's change post that itself. don't want to block this PR. font changes here can be picked up separate